institution/wagtail_hooks.py

```diff
@@ -28,4 +28,35 @@ class InstitutionAdmin(ModelAdmin):
     export_filename = 'institutions'
 
 
-modeladmin_register(InstitutionAdmin)
+class SponsorCreateView(CreateView):
+
+    def form_valid(self, form):
+        self.object = form.save_all(self.request.user)
+        return HttpResponseRedirect(self.get_success_url())
+
+
+class SponsorAdmin(ModelAdmin):
+    model = Sponsor
+    create_view_class = SponsorCreateView
+    menu_label = _('Sponsor')
+    menu_icon = 'folder'
+    menu_order = 900
+    add_to_settings_menu = False  # or True to add your model to the Settings sub-menu
+    exclude_from_explorer = False  # or True to exclude pages of this type from Wagtail's explorer view
+    list_display = ('inst_name', 'inst_acronym', 'level_1', 'level_2', 'level_3',
+                    'location', 'official', 'is_official')
+    search_fields = ('inst_name', 'inst_acronym', 'level_1', 'level_2', 'level_3',
+                    'location', 'official', 'is_official')
+    list_export = ('inst_name', 'inst_acronym', 'level_1', 'level_2', 'level_3',
+                    'location', 'official', 'is_official')
```

Comment:
os nomes deve ser idênticos aos nomes dos campos. Não há o atributo inst_name, nem inst_acronym, por exemplo.
Exceto em search_fields, que deve colocar o nome do campo e `__` se o atributo for de outro modelo.

Reply:
Ok @robertatakenaka